timescaledb/tutorials/analyze-intraday-stocks/index.md

```diff
@@ -0,0 +1,46 @@
+# Analyze historical intraday stock data
+
+This tutorial is a step-by-step guide on how to collect, store, and analyze intraday stock data 
+with TimescaleDB.
+
+We are going to follow a simple modular process:
+
+1. [Design database schema][design-schema]
+2. [Fetch and ingest stock data][fetch-ingest]
+3. [Explore stock market data][explore]
+
+First, you will create a table that will be capable of storing 1-min candlestick data. 
+
+Secondly, you will learn how to fetch data from the Alpha Vantage API and insert it into the database in a fast manner. 
+
+Finally, after all the plumbing work is done, you will see several ways to explore stock price points, lows and highs, price changes over time, symbols with the most daily gains, candlestick charts, and more!
+
+<highlight type="tip">
+Ideally, you start off with step 1 and implement each step in order to complete the full project. But you can
+also start with step 3 and start exploring a sample dataset right away.
+</highlight>
```

Comment:
```suggestion
<highlight type="tip">
Ideally, implement each step in order to complete the full project. But you can
also start exploring a sample dataset right away.
</highlight>
```

I'm not sure of the wisdom of telling people to ignore everything you've just told them, but if you want to keep this, you might need to give a hint about how to explore with setting up.

Reply:
The idea was to provide the dataset so folks can skip the first step, but it needs some more instructions than what's currently there. Let's skip this for now, and maybe come back to it later.